pkg/virtctl/adm/logverbosity/logverbosity.go

```diff
@@ -0,0 +1,451 @@
+package logverbosity
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strconv"
+
+	"github.com/spf13/cobra"
+
+	"k8s.io/client-go/tools/clientcmd"
+
+	k8smetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	v1 "kubevirt.io/api/core/v1"
+	"kubevirt.io/client-go/kubecli"
+
+	"kubevirt.io/kubevirt/pkg/apimachinery/patch"
+	virtconfig "kubevirt.io/kubevirt/pkg/virt-config"
+	"kubevirt.io/kubevirt/pkg/virtctl/templates"
+)
+
+type Command struct {
+	clientConfig clientcmd.ClientConfig
+	command      string
+}
+
+const (
+	// for command parsing
+	// try to use less weird numbers, because these numbers will be shown in the help menu
+	// there is no option to hide the default value from the help menu
+	// this is the behavior we inherit from pflag.FlagUsages, which calls the FlagUsagesWrapped function:
+	// https://github.com/kubevirt/kubevirt/blob/main/vendor/github.com/spf13/pflag/flag.go#L677
+	NoFlag = 100 // Default value if no flag is specified (dummy, we use cmd.Flags().Changed() to check if a flag is specified)
```

Comment:
Any reason to expose this const and not the other?

Reply:
There is no special reason for `100`. We can use any number between `11` and `MaxUint`. I chose a random number (100) that might be less weird than MaxUint-1 (MaxUint-1 is a very long and random number, so it looks very weird in the help menu). If you find a better number, please let me know.

I do not want to expose this number because it is irrelevant to the user. If we had an option to hide the default value from the Help menu, that would be an optimal solution. However, **there is no option to hide the default value from the Help menu**. This is the behavior we inherit from pflag.FlagUsages, which calls the FlagUsagesWrapped function:
https://github.com/kubevirt/kubevirt/blob/main/vendor/github.com/spf13/pflag/flag.go#L677

That's why this constant is exposed.

OR, there may be another workaround where we get the verbosity as a `string`, not a `uint`, as below.
```
noArg = "no arg”
noFlag = "no flag”
cmd.Flags().StringVar(&verbosities[VirtAPI], "virt-api", NoFlag, "show/set virt-api log verbosity (0-9)")
# We will need to convert verbosity from string to uint later.
```

In our earlier discussion, we decided to use `uint` instead of `string`, but at that time we did not know the behavior of the pflag package Help menu.

Even in this workaround, the default value is exposed in the Help menu, and in this case, the verbosity value is `string`, not `uint`, in the Help menu, which may be confusing to users. So I think the current workaround (uint) is better than this (string).
`--all string[=no arg] show/set all component log verbosity (0-9) (default no flag)`

Any advice would be appreciated.

Reply:
I changed NoFlag from 100 to 11, because I think the continuous number (10, 11) might be seen as less random.